internal/cli/main.go

```diff
@@ -29,6 +29,10 @@ func Run(args []string) bool {
 		migrateCmd     = app.Command("migrate", "migrate command")
 		migrateUpCmd   = migrateCmd.Command("up", "migrate db up")
 		migrateDownCmd = migrateCmd.Command("down", "migrate db down")
+
+		event    = app.Command("event", "claim event command")
+		eventCmd = event.Command("create-early-test", "claim event command")
+		data     = eventCmd.Arg("-before", "data after ...").Required().Int()
```

Comment:
Just `before`. In previous comment it was as a example, because when you pass parameters, you need use -paramname